WLC/image_processing/character.py

```diff
@@ -9,17 +9,20 @@
 
 
 class Character(ExtendedImage):
-    def __init__(self, image, x, y, w, h):
-        super().__init__(image, x, y, w, h)
+    def __init__(self, image, x_axis, y_axis, width, height, extended_image=None,
+                 show_pic=False, show_line=False, show_word=False, show_char=False):
```

Comment:
We should enclose these booleans in a class and pass that around instead.

Reply:
Doing that now, had planned to do at later date but no point delaying it